docs/topics/logging.txt

```diff
@@ -728,23 +728,57 @@ logging module.
 Django's default logging configuration
 ======================================
 
-By default, Django configures the following logging:
+By default, all Django loggers, except :ref:`django-server-logger`,
+propagate all logging calls to their parents, up to the root logger which is
+``django`` logger. On this logger, the ``console`` and ``mail_admins`` handlers
+are attached to provide the following result:
 
 When :setting:`DEBUG` is ``True``:
 
-* The ``django`` logger sends messages in the ``django`` hierarchy (except
-  ``django.server``) at the ``INFO`` level or higher to the console.
+* The root ``django`` logger sends to the console all messages of level ``INFO``
+  or higher, which propagate from its children (except ``django.server``).
```

Comment:
I think leaving discussion about propagation to a separate paragraph would be simpler. (I'm not sure what it means.)